MobileCenter/MobileCenter/Internals/Vendor/Reachability/MS_Reachability.m

```diff
@@ -66,7 +66,13 @@ @implementation MS_Reachability {
 
 + (instancetype)reachabilityWithHostName:(NSString *)hostName {
   MS_Reachability *returnValue = NULL;
-  SCNetworkReachabilityRef reachability = SCNetworkReachabilityCreateWithName(NULL, [hostName UTF8String]);
+
+  const char * utf8HostName = [hostName UTF8String];
```

Comment:
Please revert the changes in MS_Reachability.m. It's based on Apple's sample code and I'd rather not change it. You can exclude this file from this kind of warnings.
I didn't mention this in my ticket for the task, so apologies, it's my fault.